plutus-core/plutus-core/src/PlutusCore/Evaluation/Result.hs

```diff
@@ -16,10 +16,10 @@ module PlutusCore.Evaluation.Result
     , isEvaluationFailure
     ) where
 
-import PlutusPrelude
-
 import PlutusCore.Pretty
 
+import PlutusPrelude
+
```

Comment:
This is pretty much the only thing it does in modules I've written, i.e. only makes things worse. Hard for me to vote in favor of adopting it.